### PR TITLE
Issue#1415 tunnel welcome

### PR DESCRIPTION
**Pull request by jenna1k**
related #1415

e2e test needs to be done

### CODE REVIEW

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.module.ts [hunk 1]
```diff
@@ -29,6 +31,8 @@ import { MatButtonModule } from '@angular/material/button';
     MatExpansionModule,
     MatProgressSpinnerModule,
     MatButtonModule,
+    MatCardModule,
+    MatGridListModule
```
Comment (Mussuranae) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.module.ts]: You really use the grid list ?

Comment (jenna1k) [reply apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.module.ts]: removed it

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel-routing.module.ts [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
 import { MovieTunnelComponent } from "./movie-tunnel.component";
+import { StartTunnelComponent } from "./start/start-tunnel.component";
 
 const tunnelRoutes: Routes = [
   {
     path: '',
     component: MovieTunnelComponent,
+    children: [
+      {
+        path: 'start',
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel-routing.module.ts]: I would use an empty string or redirectTo 'start' when empty string is provided.

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel-routing.module.ts [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+        component: StartTunnelComponent
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel-routing.module.ts]: Use a `loadChildren: () => import('...').then(m => m.StartTunnelComponent)` instead. We don't want to load it if its not used by the user

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.module.ts [hunk 4]
```diff
@@ -15,9 +16,9 @@ import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatExpansionModule } from '@angular/material/expansion';
 import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
 import { MatButtonModule } from '@angular/material/button';
-
+import { MatCardModule } from '@angular/material/card';
 @NgModule({
-  declarations: [MovieTunnelComponent],
+  declarations: [MovieTunnelComponent, StartTunnelComponent],
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/movie-tunnel.module.ts]: With Lazy Loading strategy you don't need the StartTunnelComponent & MatCardModule. You'll need to create a module `StartTunnelModule` though

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    <h1>Add a Title and Licensed Rights</h1>
+    <h3>The 4-step process of adding a listing and distribution rights will take about a half-hour to complete. </h3>
+    <p>You can fill it out in one go or over time. You may navigate between steps, but all sections must be completed before submitting to Archipel Content for validation. </p>
+    <a mat-flat-button color="accent" routerLink="#">
+      <span>Begin</span>
+    </a>
+  </section>
+  <section fxFlex.gt-sm="45" fxLayout="row wrap" fxLayoutAlign="flex-start" fxLayoutAlign.lt-sm="center">
+    <ng-container *ngFor="let content of cardContents">
+      <mat-card fxFlex="0 1 calc(50% - 20px)">
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.html]: I would recommend to use a simple width on the css. I think it's easier to read/understand. It's up to you

File: apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
+:host {
+  display: block;
+  margin-left: 5%;
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/dashboard/movie-tunnel/start/start-tunnel.component.scss]: Never use % for margin. It'll break when you resize the page.

**Pull request merged**